playbook.md

```diff
@@ -2146,9 +2146,11 @@ https://www.aviva.co.uk/retirement/news-views/report/making-sense-of-tax-relief-
 
 #### Childcare Vouchers
 
-dxw have a Childcare vouchers scheme with [Busy Bees Benefits](www.busybeesbenefits.com/childcare)
-For more information, check BreatheHR or let Vanessa know you would like to join
-the scheme.
+The Government closed the Childcare Vouchers scheme to new entrants on 4th October 2018. This means we are no longer able to add new members to the voucher scheme with Busy Bees.
+
+The government has introduced new ways to help parents with childcare costs which you access directly: www.childcarechoices.gov.uk
```

Comment:
To keep the linter happy I would suggest the following change.
```suggestion
The government has introduced new ways to help parents with childcare costs which you access directly: [www.childcarechoices.gov.uk](www.childcarechoices.gov.uk)
```

Reply:
I think I've added this now - can you check please @rjw1?